### PR TITLE
Review my html/css popup please

### PR DESCRIPTION
**Pull request by alexey1985-FE**
### HTML & CSS practice: Hooli-style Popup

[Demo](https://alexey1985-fe.github.io/HTML-CSS-Popup/.)
[Code base](https://github.com/alexey1985-FE/HTML-CSS-Popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/alexey1985-FE/html-css-popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+            box-sizing: border-box;
+}
+body {
+    font-family: Arial, sans-serif;
+}
+.nav__wrap {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: end;
```
Comment (A-Ostrovnyy) [inline submissions/alexey1985-FE/html-css-popup/css/style.css:14]: Avoid writing properties with vendor prefixes.
Assign this to the bundler.

File: submissions/alexey1985-FE/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        <li><a class="main__menu__item" href="#">Mail</a></li>
+        <li><a class="main__menu__item" href="#">Images</a></li>
+        <li class="menu-toggle">
+              <label for="menu-toggle__input">
+                 <img class="menu-toggle__icon" src="img/popup-button.png" alt="popup">
+              </label>
+              <input class="menu-toggle__input" type="checkbox" id="menu-toggle__input">
+              <div class="sub-menu">
+                   <div class="sub-menu__container">
+                      <div class="sub-menu__list">
```
Comment (A-Ostrovnyy) [inline submissions/alexey1985-FE/html-css-popup/index.html:23]: Class name is `sub-menu__list` but HTML tag is `div`)

File: submissions/alexey1985-FE/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                            <a class="sub-menu__item" href="#">
+                                <img src="img/apps.png" alt="apps">
+                                Apps
+                            </a>
+                            <a class="sub-menu__item" href="#">
+                                <img src="img/documents.png" alt="documents">
+                                Documents
+                            </a>
+                      </div>
+                          <input class="sub-menu__input" type="checkbox" id="sub-menu__input">
```
Comment (A-Ostrovnyy) [inline submissions/alexey1985-FE/html-css-popup/index.html:73]: Fix grid inside sub-menu
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good Work!
See my comments above!

**Comment by alexey1985-FE** [comment]
Thanks for your review and advices.  I will fix it.  But it is strange that the grid went, in chrome and firefox it was all right.

**Comment by A-Ostrovnyy** [comment]
Welcome to the web development world.)
Different browsers different implementations/interpretations of the specification.

**Comment by alexey1985-FE** [comment]
It's true, but only chrome was specified in the task.  There was no cross-browser requirement for this task:)

**Comment by alexey1985-FE** [comment]
IMG_20201217_213226

**Review by A-Ostrovnyy — APPROVED** [review]
Well done

**Comment by alexey1985-FE** [comment]
thank you very much for the quick response and advices

**Pull request merged**